[blog] New posts (#22607)
mui-org/material-ui: docs/pages/blog/marija-najdova-joining.md

---
title: Marija Najdova joins Material-UI
description: We are excited to share that Marija Najdova has joined Material-UI. She has started this week full-time and is now part of the community team.
date: 2020-09-15T00:00:00.000Z
authors: ['oliviertassinari']
---

> We are excited to share that [Marija Najdova](https://twitter.com/marijanajdova) has joined Material-UI. She has started this week full-time and is now part of the community team.

Reviewer: ```suggestion
We are excited to share that [Marija Najdova](https://twitter.com/marijanajdova) has joined Material-UI. She has started this week full-time, and is now part of the community team.
```

Before joining Material-UI, Marija worked on the React implementation of [Fluent UI](https://www.microsoft.com/design/fluent/) at Microsoft. She's passionate about React, design systems, and component driven development. At Microsoft, as part of the Fluent UI core team since 2018, she was responsible for the icons, animations and various theme related features.

> I have been impressed by the dedication Marija puts into her work. She has made important changes happened even before starting, during her free time! For instance, she has recently introduced a new structure of the theme object to [add custom variants](https://next.material-ui.com/customization/components/#adding-new-component-variants) for v5:

Reviewer: Strange, we lost an edit in the last review:

```suggestion
Marija is off to a running start, having made important changes happen during her free time, even before starting. These include a new structure for the theme object, as well as the ability to [add custom variants](https://next.material-ui.com/customization/components/#adding-new-component-variants) in v5:
```

> 
> ```jsx
> const theme = createMuiTheme({
>   components: {
>     MuiButton: {
>       variants: [
>         {
>           props: { variant: 'dashed' },
>           style: {
>             textTransform: 'none',
>             border: `2px dashed grey${blue[500]}`,
>           },
>         },
>       ],
>     },
>   },
> });
> ```

Author: ```suggestion
```

> ```

Reviewer 2: +1 we already have the link in the text, if people are interested they can take a look :)

> ```

Author: My intention was to make the post more "vivid" by introducing some content that isn't just text. Code in this case.
